packages/blocks/src/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { RepositoryIcon, TrashIcon } from '@mergestat/icons';
+import React from 'react';
+
+type ListItemProps = {
+  repositoryName: string;
+  subline?: string;
+  onClick?: () => void;
+};
+
+export const ListItem: React.FC<ListItemProps> = ({
+  repositoryName,
+  subline,
+  onClick,
+}) => {
+  return (
+    <div className="w-full">
+      <div className="w-full t-list-item-container">
+        <div className="t-left-side">
+          <button className="t-list-button">
+            <RepositoryIcon className="t-icon" />
```

Comment:
let's make this icon passable as a prop, so user can supply different icons for different contexts